lib/ruby_lsp/ruby_lsp_rails/server.rb

```diff
@@ -258,8 +258,10 @@ def execute(request, params)
             send_result(run_migrations)
           end
         when "reload"
-          with_notification_error_handling(request) do
-            ::Rails.application.reloader.reload!
+          with_progress("rails-reload", "Reloading Rails application context") do
```

Comment:
```suggestion
          with_progress("rails-reload", "Reloading Rails application") do
```
just as clear?

Reply:
I was afraid that saying "Reloading application" could make the user think we're reloading the Rails server they might be running, when in fact we're reloading our own custom one.

Reply:
Maybe "Reloading the Ruby LSP Rails instance of your application"